public/content/lessons/2016/cross-products/index.mdx

```diff
@@ -1,10 +1,253 @@
+>
+
+The cross product $\vec{\mathbf{w}} \times \vec{\mathbf{v}}$ is negative since $\vec{\mathbf{v}}$ is on the left of $\vec{\mathbf{w}}$.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramQuestion2Answer.svg" 
+/>
+
+</Question>
+
+## Compute with determinant
+
+Of course, you want to be able to compute this without someone telling you the area and this is where the determinant comes in from <LessonLink id="determinant">chapter 5</LessonLink>. For the 2d cross product $\vec{\mathbf{v}}$ cross $\vec{\mathbf{w}}$, you write the coordinates of $\vec{\mathbf{v}}$ as the first column of a matrix, write the coordinates of w as the second column, then you compute the determinant.
+
+<Figure
+    image="./figures/compute-with-determinant/ComputeCrossProduct.svg" 
+/>
+
+This is because a matrix whose columns represent $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$ corresponds with a linear transformation that moves the basis vectors i hat and j hat to $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$. 
+
+<Figure
+    image="./figures/compute-with-determinant/ChangeOfArea.svg"
+    video="./figures/compute-with-determinant/ChangeOfArea.mp4"
+    show="video" 
+/>
+
+The determinant is all about measuring how areas change due to a transformation, and the prototypical area to look at is the unit square resting on i hat and j hat. After the transformation, that square turns into the parallelogram we care about. So the determinant, which generally measures the factor by which areas are changed, gives the area of this parallelogram, since it evolved from a square that started with area 1. 
+
+<Figure
+    image="figures/determinant-basis-vectors-transformed-orientation.png" 
+    caption="TODO: replace with image callback from chapter 5"
+/>
+
+What's more, if $\vec{\mathbf{v}}$ is to the left of $\vec{\mathbf{w}}$, it means orientation is flipped during the transformation, which is what it means for a determinant to be negative.
+
+<Figure
+    image="./figures/compute-with-determinant/DeterminantNegative.svg"
+    video="./figures/compute-with-determinant/DeterminantNegative.mp4"
+    caption="Since the orientation is flipped, the determinant is negative."
+/>
+
+<Figure
+    image="./figures/compute-with-determinant/DeterminantPositive.svg"
+    video="./figures/compute-with-determinant/DeterminantPositive.mp4"
+    caption="Since the orientation isn't flipped, the determinant is positive."
+/>
+
+### Determinant example
+
+<Figure
+    image="./figures/determinant-example/DeterminantExample.svg" 
+/>
+
+As an example, if $\vec{\mathbf{v}}$ has coordinates $\begin{bmatrix}-3\\1\end{bmatrix}$, and $\vec{\mathbf{w}}$ has coordinates  $\begin{bmatrix}2\\1\end{bmatrix}$, the determinant of the matrix with those coordinates has columns is $(-3)\cdot1 - 2\cdot1$, which is $-5$. So evidently the area of the parallelogram they define is $5$, and since $\vec{\mathbf{w}}$ is to the right of $\vec{\mathbf{v}}$ it should make sense that this value is negative.
```

Comment:
Instead of the phrase "the determinant of the matrix...", just write this as an equation in a line below.